### DISCUSSION TITLE
Does it have asymmetrical tiling ?

### DISCUSSION THREAD

**Discussion by jony100200** [Stability-AI/StableSwarmUI#411] [General]
I use a111 mostly due to the fact it has tiling. Does this have tiling too? Can I tile only on y or x or both if needed? Are they seamlessly tileable?
I use this addon with A111

https://github.com/tjm35/asymmetric-tiling-sd-webui

**Comment by mcmonkey4eva** [comment]
Yep! Under "Sampling" parameters, find "Seamless Tileable" and select the timing mode you want
![image](https://github.com/Stability-AI/StableSwarmUI/assets/4000772/af84673c-987e-4a48-9a76-612cda385c08)
